main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 	insecureSkipVerify := flag.Bool("skip-registry-cert-verification", false, "whether to skip registries' certificate verification")
 	defaultRegistry := flag.String("default-registry", "",
 		fmt.Sprintf("default registry to use in absence of a fully qualified image name, defaults to %q", name.DefaultRegistry))
+	checkDisabled := flag.Bool("check-disabled", false, "whether to force image check for disabled / suspended workloads")
```

Comment:
Boolean flags are not extensible.
1. I was puzzled by the name of the flag and dived into a code to understand what is going on. It would be better to name the flag something like `force-check-disabled-controllers`.
1. Let’s make it a white-list of kinds to force check if they are disabled like `—force-check-disabled-controllers=DaemonSet`, or `*` for all kinds.

Reply:
Ok, I will try to implement the feature as you described